mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -244,7 +244,12 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_5634 getSoundCategory ()Lnet/minecraft/class_3419;
 	METHOD method_5636 setYaw (F)V
 		ARG 1 yaw
-	METHOD method_5637 isTouchingWater ()Z
+	METHOD method_5637 isWet ()Z
+		COMMENT Returns whether this entity is touching water or is being rained on or is inside a bubble column.
```

Comment:
The way this is said implies the list is exhaustive. You could use `...` or `eg.` to take into account mods or future updates.

Reply:
That's true, also made the changes then.